tests/conftest.py

```diff
@@ -81,14 +113,24 @@ def _app_factory(config=None):
         SERVER_NAME="example.com",
         TESTING=True,
         WTF_CSRF_ENABLED=False,
+        ACCOUNTS_SITENAME="invenio",
+        ACCOUNTS_BASE_TEMPLATE="invenio_accounts/base.html",
+        ACCOUNTS_SETTINGS_TEMPLATE="invenio_accounts/settings/base.html",
+        ACCOUNTS_COVER_TEMPLATE="invenio_accounts/base_cover.html",
+        WEBPACKEXT_MANIFEST_LOADER=MockManifestLoader,
```

Comment:
i tested it, and None is not enough because of following error message: ` AttributeError: 'NoneType' object has no attribute 'split'`. i didn't tested if every variable has this problem, but to be consistent i would set all of them.